tools/monorepo-utils/src/code-freeze/commands/version-bump/lib/update.ts

```diff
@@ -30,8 +30,12 @@ export const updateReadmeChangelog = async (
 		);
 
 		await writeFile( filePath, updatedReadmeContents );
-	} catch ( e ) {
-		Logger.error( e );
+	} catch ( e: unknown ) {
```

Comment:
This pattern is pretty common, to avoid `any` and because of how JS works you can throw literally anything whence the `unknown`. It's verbose I know, but it's TS trying to be type saf(er) in an environment where anything could be thrown.

We check if its an `instanceof` Error and deal with it appropriately.